static/includes/config-mysql.md

```diff
@@ -298,7 +298,7 @@
 
 **Title:** innodb_flush_neighbors
 
-**Description:** Specifies whether flushing a page from the InnoDB buffer pool also flushes other dirty pages in the same extent (default is 1): 0 - dirty pages in the same extent are not flushed,  1 - flush contiguous dirty pages in the same extent,  2 - flush dirty pages in the same extent
+**Description:** Specifies whether flushing a page from the InnoDB buffer pool also flushes other dirty pages in the same extent (default is 1): 0 - dirty pages in the same extent are not flushed, 1 - flush contiguous dirty pages in the same extent, 2 - flush dirty pages in the same extent
```

Comment:
This file is automatically generated so fixing this will be reverted on the 1st next month 🙃 
This should be fixed in `mysql/service_user_config.py` in the monorepo 👍